mapbox_tilesets/scripts/cli.py

```diff
@@ -799,3 +800,78 @@ def estimate_area(features, precision, no_validation=False, force_1cm=False):
             }
         )
     )
+
+
+@cli.command("list-activity")
+@click.argument("username", required=True, type=str)
+@click.option(
+    "--sortby",
+    required=False,
+    type=click.Choice(["requests", "modified"]),
+    default="requests",
+    help="Sort the results by request count or modified timestamps (default: 'requests')",
+)
+@click.option(
+    "--orderby",
+    required=False,
+    type=click.Choice(["asc", "desc"]),
+    default="desc",
+    help="Order results by asc or desc for the sort key (default: 'desc')",
+)
+@click.option(
+    "--limit",
+    required=False,
+    type=click.IntRange(1, 500),
+    default=100,
+    help="The maximum number of results to return, from 1 to 500 (default: 100)",
+)
+@click.option(
+    "--start",
+    required=False,
+    type=str,
+    help="Pagination key from the `next` value in a response that has more results than the limit.",
+)
+@click.option("--token", "-t", required=False, type=str, help="Mapbox access token")
+@click.option("--indent", type=int, default=None, help="Indent for JSON output")
+def list_activity(
+    username,
+    sortby=None,
+    orderby=None,
+    limit=None,
+    start=None,
+    token=None,
+    indent=None,
+):
+    """List tileset activity for an account. Response is an ordered array of aggregate resource data.
```

Comment:
I'm not sure "an ordered array of aggregate resource data" will make sense to users. Could we say what it's ordered by, and then just turn "aggregate resource data" into plainer language? i.e. sum of requests over the past 30 days.